sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala

```diff
@@ -419,33 +419,47 @@ abstract class RapidsShuffleInternalManagerBase(conf: SparkConf, val isDriver: B
   }
 }
 
+/**
+ * Trait that makes it easy to check whether we are dealing with the
+ * a RAPIDS Shuffle Manager
+ *
+ * TODO name does not match its function anymore
```

Comment:
Are you going to fix this TODO?

Reply:
I filed https://github.com/NVIDIA/spark-rapids/issues/3624 for this.